content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -926,9 +926,9 @@ prisma.log.create({
 
 <Admonition type="info">
 
-These null enums do not apply to MongoDB because there is not a difference between a JSON null and a database NULL.
+These _null enums_ do not apply to MongoDB because there the difference between a JSON `null` and a database `NULL` does not exist.
 
-They also do not apply to the `array_contains` operator becuase there can only be a JSON null within an JSON array. Since there cannot be a database NULL within a JSON array, `{ array_contains: null }` is not ambiguous.
+They also do not apply to the `array_contains` operator in all databases because there can only be a JSON `null` within a JSON array. Since there cannot be a database `NULL` within a JSON array, `{ array_contains: null }` is not ambiguous.
 
```

Comment:
@janpio it doesn't seem clear what is meant by ".. is not ambiguous." Does it mean "it is not ambiguous, i.e. it is quite clear that null values will indeed be allowed?" I thin we need to work on that wording. 
Also, what is the INCLUDE_NULL_VALUES option... does that need to be mentioned??

Reply:
That is existing content that I did not touch (so out of scope of this change and Pull Request). 

@millsp is possibly the better person to answer, but if I had to explain I would say that `null` only has one possible meaning which types of the two null it means.

Let's just merge this though please if this requires a bigger rewrite and do that separately.

Reply:
It means that it is not ambiguous to the database in that very specific case. Because you're already in a JSON data structure, it is clear that it is the json `null` and not the DB `NULL`.

This `null` distinction comes handy when you have a json field that is nullable. That means your field can be `NULL` (db) or `null` (json), depending on what you chose to insert (or not). This is useful to actually find fields that are not set vs fields that are just empty json.

Reply:
Does this clarify things @tanberry?

Reply:
Yes, thanks!